programs/scholarship.md

```diff
@@ -96,6 +92,12 @@ The following documents are provided for your convenience.
       <td>University of California, Merced</td>
       <td>2016</td>
     </tr>
+    <tr>
+      <td>Javier Fuentes</td>
+      <td>Aspire California Prepatory</td>
+      <td>University of California, Davis</td>
+      <td>2017</td>
```

Comment:
What do you think about naming this column *Award Year*?

Reply:
*shrug*. Works for me.